sources/packages/web/src/views/aest/student/StudentDetails.vue

```diff
@@ -23,7 +23,7 @@
       </header-navigator>
     </template>
     <template #tab-header>
-      <v-tabs :model="tab" stacked color="primary"
+      <v-tabs :model="tab" stacked color="primary" class="mt-n14"
```

Comment:
Can you please elaborate on why this class was needed? `class="mt-n14"`

Reply:
I used this class to move the tabs up, thus reducing the space between the `full-page-container` subtitle and the `tabs-header`

Reply:
I got the point about why it was done, I was wondering if it needed to be done with a negative margin. IMO, applying a negative can be used when there is no alternative way. For instance, maybe there is something else to be adjusted that is actually causing the additional spacing.

Reply:
For this one, `full-page-container` has `layoutTemplate` with a `div` that adds a margin `mt-4`.

Reply:
As we talked over the call, lets check if it is possible to have the `full-page-container` updated to the `centered-tab` template.

Reply:
Updated `full-page-container` to remove the extra padding and the margin from the CenteredTab layout and verified that it impacts the components - StudentDetails, InstitutionDetails, LocationActiveApplicationSummary and LocationCEOSummary and improves them by removing the extra white space.